src/pages/SignIn.js

```diff
@@ -22,11 +20,6 @@ const styles = {
 const SignIn = () => {
   return h(HeroWrapper, { showMenu: false }, [
     div({ style: { maxWidth: 600 } }, [
-      div({ style: { fontSize: 36, color: colors.dark(0.6) } }, ['New User?']),
-      div({ style: { fontSize: 36, marginBottom: '2rem' } }, [`${getAppName()} requires a Google Account.`]),
```

Comment:
Removed this text per https://miro.com/app/board/uXjVOZMbcqQ=/

(Soon it won't be true)

Reply:
Added it back